specificatie/verblijfplaats.yaml

```diff
@@ -8,23 +8,27 @@ components:
   schemas:
     Adresregel1:
       type: string
+      pattern: ^[a-zA-Z0-9À-ÿ .\'\-\']{1,100}?$
```

Comment:
ook hier hoort de . voorafgegaan door een \
 en is 1x \' voldoende

is dit niet te strikt is voor een locatiebeschrijving en buitenlands adres? Kan daar bijvoorbeeld º, een komma of andere leestekens in staan?

Reply:
hoe kom je aan maximale lengte 100?

Regel 1 adres buitenland: 35
Adres: 24 (korteNaam) + 1 + 5 (huisnummer) + 1 (huisletter) + 1 + 4 (toevoeging) + 1 + 3 (aanduiding) = 40
Locatie: 35

Reply:
maximale lengte is voor gebruikers belangrijk, want de adresregels moeten in een vensterenvelop of op een brief passen.